linux/flatpak/flatpak-libnm.yaml

```diff
@@ -14,64 +14,57 @@ modules:
 
   - name: libndp
     buildsystem: autotools
-    sources:
-      - type: archive
-        url: http://libndp.org/files/libndp-1.6.tar.gz
-        sha256: 0c7dfa84e013bd5e569ef2c6292a6f72cfaf14f4ff77a77425e52edc33ffac0e
-
-  - name: udev
-    config-opts:
-      - --disable-hwdb
-      - --disable-logging
-      - --disable-gudev
-      - --disable-introspection
-      - --disable-keymap
-      - --disable-mtd_probe
     cleanup:
```

Comment:
What do the changes in this file have to do w/ the Flatpak assets for flathub?

Reply:
The `NetworkManager` changes are not required for updating the `org.mozilla.vpn.metainfo.xml` manifest - I can pull them out and address this later.

Where the change came from: In my travels, I had noticed that Surfshark was able to build the `libnm` library without needing to build udev so I used their flatpak as a learning excercise in how to remove some unnecessary cruft from our build and minimize the dependencies.

Reply:
I think changing our sources should be a different PR, as that seems like a potentially risky change that should be thought about separately.